nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestPutUDP.java

```diff
@@ -190,15 +191,17 @@ public void testLoadTest() throws Exception {
         checkInputQueueIsEmpty();
     }
 
-    private void reset(final String address, final int port, final int recvQueueSize) throws Exception {
+    private void reset(final String address, final int port, final int frameSize) throws Exception {
         runner.clearTransferState();
         removeTestServer();
-        createTestServer(address, port, recvQueueSize);
+        createTestServer(address, port, frameSize);
     }
 
     private void configureProperties(final String host, final boolean expectValid) {
         runner.setProperty(PutUDP.HOSTNAME, host);
-        runner.setProperty(PutUDP.PORT, Integer.toString(server.getLocalPort()));
+        runner.setProperty(PutUDP.PORT, Integer.toString(port));
+        runner.setProperty(PutUDP.MAX_SOCKET_SEND_BUFFER_SIZE, "5 MB");
```

Comment:
Is there a reason for setting `5 MB` here?  That seems larger than theoretically possible given the maximum size of UDP packets.